### PR TITLE
DSP-184, 185, 186

### PR DESCRIPTION
**Pull request by JoeChapman**
- check-details
- help with form
- confimation

- add apt templates update others
- remove uk-address page and update contextual content
- amend and add apt content/template strings
- add emails for collection journey

### CODE REVIEW

File: controllers/collection-check-details.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+var _ = require('underscore');
+
+var CollectionCheckDetails = function CollectionCheckDetails() {
+  CheckDetails.apply(this, arguments);
+};
+
+util.inherits(CollectionCheckDetails, CheckDetails);
+
+function getReason(req) {
+  if (req.form.values['reason-radio'] === 'under-age') {
```
Comment (easternbloc) [inline controllers/collection-check-details.js]: :ok_hand:

File: assets/scss/app.scss [hunk 2]
```diff
@@ -103,18 +103,20 @@ h1 {
...
-  td:nth-child(1) {
-    width: 20%;
-  }
-  td:nth-child(2) {
-    width: 60%;
-  }
-  td:nth-child(3) {
-    text-align: right;
-    padding-right: 0;
+  tbody {
```
Comment (easternbloc) [inline assets/scss/app.scss]: Do we need this for the thead?

File: controllers/collection-check-details.js (hunk 1 above)
Comment (JoeChapman) [inline controllers/collection-check-details.js]: is that a rabbit hand?

File: assets/scss/app.scss (hunk 2 above)
Comment (JoeChapman) [inline assets/scss/app.scss]: not for now because all our thead's are set to colspan=3

File: controllers/collection-check-details.js (hunk 1 above)
Comment (easternbloc) [inline controllers/collection-check-details.js]: it's a pukka hand.

File: assets/scss/app.scss (hunk 2 above)
Comment (easternbloc) [inline assets/scss/app.scss]: fair

**Pull request merged**